Automattic/wp-calypso
# Fix track event flow name in signup page in sensei flow, update step name of launch

Related to https://github.com/Automattic/wp-calypso/issues/84153

## Proposed Changes

* When user is not logged in and clicks continue in senseiSetup page (`http://calypso.localhost:3000/setup/sensei/sensei-setup`), user is taken to the signup/login page. Even though they were sent there from sensei, the events get logged with flow value 'account', we've fixed that here
* Updated sensei-launch step's stepName because it was saying senseiSetup, it shouldn't have any other implication than in the event log https://github.com/Automattic/wp-calypso/pull/84191/commits/d6c14db2cc1edfead65d5ad9117b91425ca3b87f

## Testing Instructions



1. Add a debugging step at https://github.com/Automattic/wp-calypso/blob/a7c2207d0929c98991013206153a6277e747a6a1/client/lib/analytics/tracks.js#L11
2. Start at http://calypso.localhost:3000/setup/sensei/senseiSetup.
3. Enter a _Site name_ and click _Continue_.
4. Output the `eventName` and `eventProperties` to the console when the debugger is triggered.
5. Make sure that `flow` is set to `senesi`, not `account` as it says in the issue.
6. ------ For steps related changes ------
7. Complete the whole flow, meaning all the steps of onboarding. Make sure everything is working as expected

## Pre-merge Checklist
